docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc

```diff
@@ -0,0 +1,171 @@
+:toc: macro
+== 32.03 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|32.02.127 TODO:
+|Code name
+|O'Neal - Update 2, 32.02 TODO:
+
+|Release date
+|January 28th, 2024 TODO:
+
+|Type
+|Maintenance Release
+
+|SHA-256 TODO:
+|a4790f3e94509fb1d80b0ff5a3567ac57295fd52b976b7f67d34ab268f9636ee
+
+|===
+
+Review the https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements] to learn about the supported operating systems, hypervisors, runtime, tools, and orchestrators.
```

Comment:
Please check the https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/install/system_requirements link, it throws 404 for me. I am not sure if I have insufficient permissions. Please double-check.

Reply:
@smitapaloalto 
It appears that the redirect is broken. We need to change it to https://docs.prismacloud.io/en/compute-edition/32/admin-guide/install/system-requirements
Can you confirm that the link works?

Reply:
The link works fine now. thanks.

Reply:
It works fine. Thanks.